resources/assets/build/webpack.config.js

```diff
@@ -127,31 +126,26 @@ let webpackConfig = {
     ],
     enforceExtension: false,
   },
-  resolveLoader: {
-    moduleExtensions: ['-loader'],
-  },
   externals: {
     jquery: 'jQuery',
   },
   plugins: [
-    new CleanPlugin([config.paths.dist], {
-      root: config.paths.root,
-      verbose: false,
+    new ESLintPlugin({
+      failOnWarning: false,
+      failOnError:   true,
     }),
-    /**
-     * It would be nice to switch to copy-webpack-plugin, but
-     * unfortunately it doesn't provide a reliable way of
-     * tracking the before/after file names
-     */
-    new CopyGlobsPlugin({
-      pattern: config.copy,
-      output: `[path]${assetsFilenames}.[ext]`,
-      manifest: config.manifest,
+    new CleanWebpackPlugin({
+      cleanOnceBeforeBuildPatterns: [config.paths.dist],
+      verbose: false,
     }),
-    new ExtractTextPlugin({
-      filename: `styles/${assetsFilenames}.css`,
-      allChunks: true,
-      disable: (config.enabled.watcher),
+    new CopyPlugin({
+      patterns: [
+        {
+          from: config.copy,
+		  noErrorOnMissing: true,
+          to: `[path]${assetsFilenames}.[ext]`,
```

Comment:
in case of moving to [copy-webpack-plugin@8.0.0](https://github.com/webpack-contrib/copy-webpack-plugin/releases/tag/v8.0.0):

> [ext] doesn't require . (dot) before, i.e. change [name].[ext] to [name][ext]

```suggestion
          to: `[path]${assetsFilenames}[ext]`,
```

Reply:
https://github.com/roots/sage/pull/2607/commits/37c000596314184002d6ccc3afdb90da8abbce29